assets/src/edit-story/components/panels/design/animation/effectInput.js

```diff
@@ -20,31 +20,68 @@
 import { useCallback } from 'react';
 import PropTypes from 'prop-types';
 import { v4 as uuidv4 } from 'uuid';
-import { rgba } from 'polished';
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 
 /**
  * Internal dependencies
  */
+import { __ } from '@web-stories-wp/i18n';
 import { FIELD_TYPES } from '../../../../../animation';
 import { GeneralAnimationPropTypes } from '../../../../../animation/outputs';
 import { AnimationFormPropTypes } from '../../../../../animation/types';
-import { DropDown, BoxedNumeric } from '../../../form';
 import RangeInput from '../../../rangeInput';
+import {
+  NumericInput,
+  DropDown,
+} from '../../../../../design-system/components';
+import {
+  THEME_CONSTANTS,
+  Text,
+  ThemeGlobals,
+} from '../../../../../design-system';
 import { DirectionRadioInput } from './directionRadioInput';
+import { INPUT_HEIGHT } from './constants';
 
 const RangeContainer = styled.div`
   width: 100%;
 `;
 
+const outerGridBordersCss = css`
+  border-radius: 0;
+  border-color: ${({ theme }) => theme.colors.border.defaultNormal} transparent
```

Comment:
This is basically hiding the outer borders inside the grid so that the wrapper of the grid could have border around it with radius in the each corner -- using the input borders with border-radius would get a bit tricky here since the number of inputs can change.